src/init.cpp

```diff
@@ -286,15 +288,13 @@ void Shutdown(InitInterfaces& interfaces)
     }
 #endif
 
-#ifndef WIN32
     try {
         if (!fs::remove(GetPidFile())) {
             LogPrintf("%s: Unable to remove PID file: File does not exist\n", __func__);
         }
     } catch (const fs::filesystem_error& e) {
         LogPrintf("%s: Unable to remove PID file: %s\n", __func__, e.what());
```

Comment:
nit: Use `fsbridge::get_filesystem_error_message(e)` instead of `e.what()`

Reply:
Ok, can be added. Just note this was not added/modified by me and was merged in https://github.com/bitcoin/bitcoin/pull/15278 a few days ago

Reply:
That was OK to use `e.what()` in other OS except Windows. So you have to use `fsbridge::get_filesystem_error_message(e)` instead.